AppBuilder/platform/ABDataCollection.js

```diff
@@ -402,11 +334,11 @@ module.exports = class ABDataCollection extends ABDataCollectionCore {
     * this alerts us of a change in our data that came from a remote
     * source: socket update, Relay response, etc...
     */
-   remoteUpdate(data) {
-      super.remoteUpdate(data).then(() => {
-         // make sure local storage has these values in it:
-         return this.datasource.model().local().localStorageStore(data);
-      });
+   async remoteUpdate(data) {
+      await super.remoteUpdate(data);
+
+      // make sure local storage has these values in it:
+      await this.datasource.model().local().localStorageStore(data);
```

Comment:
I didn't like this until I checked `local` : it does an `updateNewer()` check on the data. This function does as the comment says "alerts us of a change" and lets local decide what to do with it. This is an example of what I have *usually* expected to happen

Reply:
I just wanna paste the link
https://vscode.dev/github/appdevdesigns/appbuilder_platform_mobile/blob/is/update/AppBuilder/platform/ABModelLocal.js#L202